content/concept-amendments.md

```diff
@@ -147,7 +145,7 @@ A transaction remains in the queue until one of the following happens:
 |--------------|--------|
 | 740352F2412A9909880C23A559FCECEDA3BE2126FED62FC7660D628A06927F11 | TBD |
 
-Replaces the payment processing engine with a more robust and efficient rewrite called the Flow engine. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding. This Amendment was created to replace the [FlowV2](#flowv2) amendment when a critical bug was found in the FlowV2 amendment.
+Replaces the payment processing engine with a more robust and efficient rewrite called the Flow engine. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding. This Amendment supersedes the [FlowV2](#flowv2) amendment.
```

Comment:
I'd still vote for breaking up the second sentence. But I realize now that that might require more discussion of floating point rounding than is feasible at the moment. We should probably document that somewhere since it has caused some confusion, but I think this is OK for now.